deploy/k8s/minio.yml

```diff
@@ -35,7 +35,6 @@ metadata:
   name: minio
   namespace: faasm
 spec:
-  type: ClusterIP
```

Comment:
Removed these to let the cluster decide on its appropriate default service type.